src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2110,82 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        var strength = Constants.CELL_BASE_THRUST + Constants.CELL_BASE_THRUST * (HexCount/2);
+        var maxVelocity = 5.0f;
```

Comment:
This is not used before l.2137, and you have `velocity` defined l.2135, so you might want to just move this definition a few lines down to be along side velocity.